hibernate-core/src/main/java/org/hibernate/tool/schema/internal/AbstractSchemaMigrator.java

```diff
@@ -439,11 +439,23 @@ protected void applyForeignKeys(
 		}
 	}
 
-	private ForeignKeyInformation findMatchingForeignKey(ForeignKey foreignKey, TableInformation tableInformation) {
```

Comment:
The function is only used for true/false detection. So a boolean return value seems more logical.